ts/Core/Series/Series.ts

```diff
@@ -4894,6 +4897,10 @@ class Series {
             inverted = chart.inverted,
             seriesClipBox = this.clipBox,
             clipBox = this.getClipBox(animation),
+            tempClipKey = ( // #4406
+                this.sharedClipKey &&
+                this.sharedClipKey.split(',')[0] === '_temporaryClip'
+            ) ? this.sharedClipKey : void 0,
```

Comment:
I think this could be moved below the `getSharedClipKey` call and simplified to something like `isTemp = sharedClipKey.slice(0, 5) === '_temp'`.